vendor/engines/sanger_sequencing/app/views/sanger_sequencing/admin/shared/_sidenav.html.haml

```diff
@@ -1,2 +1,3 @@
 %ul.sidebar-nav
   = tab SangerSequencing::Submission.model_name.human(count: 2), facility_sanger_sequencing_admin_submissions_path(current_facility), sidenav_tab == "submissions"
+  = tab SangerSequencing::Batch.model_name.human(count: 2), facility_sanger_sequencing_admin_batches_path(current_facility), sidenav_tab == "batches"
```

Comment:
With training requests we put a count in the sidenav. Would something like that be useful for submissions or batches?